docs/clipboard-unsanitized/explainer.md

```diff
@@ -21,40 +23,54 @@ If we use the built-in sanitizer that produces an HTML fragment, the styles that
 *   Drag-and-Drop APIs.
 
 ## Additional Background
```

Comment:
This sections sounds somewhat redundant with the introduction. Can they be merged?

Reply:
Removed redundant info and merged into the intro section. I kept this section just for all the read API examples.